StudioCore/MapStudioNew.cs

```diff
@@ -637,28 +662,30 @@ private void Update(float deltaseconds)
                         }
                         ImGui.EndMenu();
                     }
-                    if (ImGui.MenuItem("Save", "CTRL+S") || InputTracker.GetControlShortcut(Key.S))
+
+                    string focusType = "";
+                    if (_msbEditorFocused)
                     {
-                        _projectSettings.Serialize(CFG.Current.LastProjectFile); //Danger zone assuming on lastProjectFile
-                        SaveParamStudioConfig();
-                        if (_msbEditorFocused)
-                        {
-                            _msbEditor.Save();
-                        }
-                        if (_modelEditorFocused)
-                        {
-                            _modelEditor.Save();
-                        }
-                        if (_paramEditorFocused)
-                        {
-                            _paramEditor.Save();
-                        }
-                        if (_textEditorFocused)
-                        {
-                            _textEditor.Save();
-                        }
+                        focusType = "Maps";
+                    }
+                    else if (_modelEditorFocused)
+                    {
+                        focusType = "Models";
+                    }
+                    else if (_paramEditorFocused)
+                    {
+                        focusType = "Params";
+                    }
+                    else if (_textEditorFocused)
+                    {
+                        focusType = "Text";
                     }
-                    if (ImGui.MenuItem("Save All", "CTRL+SHIFT+S") || ((InputTracker.GetKey(Key.ShiftLeft) || InputTracker.GetKey(Key.ShiftRight)) && InputTracker.GetControlShortcut(Key.S)))
+
+                    if (ImGui.MenuItem($"Save {focusType}", "Ctrl+S") || InputTracker.GetControlShortcut(Key.S))
```

Comment:
I think a general "save active editor" would be better, since we will eventually go to a model where editors are instantiated.

Reply:
I don't think "Save active editor" will be clear enough for the user, at least based on confusion I've seen expressed by folk in the past. I don't know what future plans entail, but is this not acceptable at the current stage? Does more good than bad, I'd say.